ci-operator/step-registry/rosa/conf/idp/htpasswd/delete/rosa-conf-idp-htpasswd-delete-ref.metadata.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
The Owners file should inherit from parent folder if I didn't make a mistaken. Please check other OWNERS file.

Reply:
I checked others. I found the Owners file exists in each level of the folder.And the precheck job will fail if no OWNERS file